staging/src/k8s.io/apiextensions-apiserver/pkg/controller/apiapproval/apiapproval_controller.go

```diff
@@ -0,0 +1,266 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos - file modified.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Multi-tenancy change is needed for this controller, otherwise it won't work on non-system-tenant CRDs. If you don't feel comfortable with this work, you can open an issue to me (@qchen) and I can pick it up when I have some free bandwidth. 

It would be nice if you can provide an e2e scenario and test data to verify the controller working.

Reply:
checked bunch of information for this protected community group, This is only for an API in *.k8s.io or *.kubernetes.io is or is not approved. For CRDs outside those groups, this doesn't applied. we don't need multi-tenancy for this.
// See kubernetes/enhancements#1111 for more details.